### ISSUE TITLE
Add side-panel file explorer view of assets-folder with drag and drop capabilities

### ISSUE THREAD

**Issue by rbosh** [glushchenko/fsnotes#1498]
**Describe your feature request**
While editing a note, it would be great have a file explorer view of the respective ./assets folder.
It could be a very basic navigable list in a collapsable side panel on the right of the main note editing view.
One could drag and drop files from the macOS DE into the file explorer view to add files to the assets folder without automatically creating a link in the note body.
One could drag and drop files from the side panel into the note body to manually create links to files.
This could disentagle the text contents from the assets folder.

**Additional context**
There are situations when I manually add files to the assets folder: for example, when I do not want to mention the file in the note (yet). For example, this could be a folder of many photos.
It would be nice to just drag and drop the file into the file explorer side panel and have it moved to ./assets without a link automatically appearing.
It is confusing that deleting the link also deletes the file from ./assets at the same time.

When I later decide to add a link to some file in the assets folder (that I moved there myself, e.g. a folder of photos – there's no way to drop a folder into a note), then there is currently no way to easily do so (other than just typing the link manually), because links do not autocomplete.
It would be great to just drag and drop the file from the asset folder view into the note and have a link be created.

In summary, this feature would untangle the contents of the text from the contents of the assets folder.
It is confusing that deleting a link from the note deletes the file since there are ways to add files to the assets folder without having a link in the note be created.
